mne_bids/write.py

```diff
@@ -965,21 +967,32 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     _mkdir_p(os.path.dirname(bids_fname))
 
     if verbose:
-        print('Copying data files to %s' % bids_fname)
-
-    if ext not in ALLOWED_EXTENSIONS:
-        raise ValueError('ext must be in %s, got %s'
-                         % (''.join(ALLOWED_EXTENSIONS), ext))
+        print('Copying data files to %s' % op.splitext(bids_fname)[0])
 
+    convert = ext not in ALLOWED_EXTENSIONS[kind]
     # Copy the imaging data files
-    if ext in ['.fif']:
+    if convert:
+        if kind == 'meg':
+            raise ValueError('Got file extension %s for MEG data, ' +
+                             'expected one of %s' % ALLOWED_EXTENSIONS['meg'])
+        if verbose:
+            warn('Converting data files to BrainVision format')
+        check_version('pybv', '0.2')
```

Comment:
I don't think you're using this correctly. Not sure what you want to do with the code here but `check_version` just returns a boolean and does not throw an error if `pybv` is not installed.